Hunting Queries/SlackAudit/SlackAuditFailedLoginsUnknownUsername.yaml

```diff
@@ -0,0 +1,20 @@
+id: d61ba8f2-e0fa-44ca-b9b9-8f938ebea98c
+name: SlackAudit - Failed logins with unknown username
+description: |
+  'This query shows attempts of failed logins with unknown username.'
+tactics: CredentialAccess
+relevantTechniques: T1110
+query: |
+  let lbtime = 24h;
+  let lbperiod = 60d;
```

Comment:
60d is not recommended, 30d max is recommended for lookback due to potential perf issues.

Reply:
Fixed, Pls check